libmscore/stafftype.cpp

```diff
@@ -236,6 +237,8 @@ void StaffType::write(XmlWriter& xml) const
             xml.tag("fretFontName",     _fretFonts[_fretFontIdx].displayName);
             xml.tag("fretFontSize",     _fretFontSize);
             xml.tag("fretFontY",        _fretFontUserY);
+            if (_genKeysig)
+                  xml.tag("keysig",     _genKeysig); // tab staff doesn't have key signature by default
```

Comment:
I guess we should better disallow key signatures on tablatures at all, it doesn't really make sense for them (at least definitely not in the way it is drawn currently). So there seems to be no much point in allowing to write such a staff type option to file for tablatures.

Reply:
OK. But that also means we should disable the "Create key signature" option in staff type change element applied to a TAB staff.

Reply:
Yes, that would be good to do indeed.